### ISSUE TITLE
End of game Tie handling

### ISSUE THREAD

**Issue by dylanpiera** [dylanpiera/Mafia-Quickplay-DiscordBot#42]
While testing today, Lone brought up a good point
if Doctor & Mafioso survive it should be a tie.

**Comment by AussieEevee** [comment]
I would say this should depend on other factors... For example, if the remaining mafia player is a role blocker, strongman, etc then the doctor's power is useless.

**Comment by dylanpiera** [comment]
Of course it would be a huge list of checks that would be required to be done.
As at the moment roleblocker ain't a thing yet, it's easy to implement. Later that'll be different.

**Comment by AussieEevee** [comment]
Is it possible to perform a check for a role that doesn't actually exist yet? Just so you don't have to go back and implement the check for a roleblocker later :p

**Comment by dylanpiera** [comment]
Yes I can do that, as Miller already has been implemented even though the role doesn't excist.
